app/javascript/packs/application.js

```diff
@@ -3,19 +3,20 @@
 // a relevant structure within app/javascript and only use these pack files to reference
 // that code so it'll be compiled.
 
-require('@rails/ujs').start();
-require('turbolinks').start();
-require('@rails/activestorage').start();
-require('channels');
+require("@rails/ujs").start()
+require("turbolinks").start()
```

Comment:
Why do we need turbolinks here?

Reply:
This appears to be generator output from running `rails new <app name>` for Rails 6

Reply:
As in, it wasn't intentionally put here